### ISSUE TITLE
Virtual Table: error when trying to create or update a record

### ISSUE THREAD

**Issue by staticset** [MicrosoftDocs/powerapps-docs#2964]
Hi.
I've tried using virtual tables. Configured according to the documentation step by step. However, I get an error when I try to create or update a record. 

Error:
Exception Message: APIM request was not successful : BadRequest : A value must be provided for item.

ErrorCode: -2147220956
HexErrorCode: 0x80040224

ErrorDetails: 
	ApiExceptionSourceKey: Plugin/Microsoft.Xrm.DataProvider.Connector.Plugins.ConnectorUpdatePlugin
	ApiStepKey: 2cdbc226-e772-ec11-8943-000d3a2b65d5
	ApiDepthKey: 1
	ApiActivityIdKey: 859d7b8f-4b7b-41d2-93c7-02e0107d92fe
	ApiPluginSolutionNameKey: msft_ConnectorProvider
	ApiStepSolutionNameKey: msft_ConnectorProvider
	ApiExceptionCategory: ClientError
	ApiExceptionMessageName: IsvUnExpected
	ApiExceptionHttpStatusCode: 400

HelpLink: http://go.microsoft.com/fwlink/?LinkID=398563&error=Microsoft.Crm.CrmException%3a80040224&client=platform

TraceText: 
	[Microsoft.Xrm.DataProvider.Connector.Plugins: Microsoft.Xrm.DataProvider.Connector.Plugins.ConnectorUpdatePlugin]
	[2cdbc226-e772-ec11-8943-000d3a2b65d5: External plug-in implementation]

Идентификатор действия: 958ce5ec-9172-45f3-a330-74456c19ae41

**Comment by Mattp123** [comment]
@staticset - Which article are you referring to for you issue? [OData v4 Data Provider configuration, requirements, and best practices](https://docs.microsoft.com/en-us/powerapps/maker/data-platform/virtual-entity-odata-provider-requirements)?

**Comment by staticset** [comment]
@Mattp123 No. [Create virtual tables using the virtual connector provider](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/virtual-entities/create-virtual-tables-using-connectors)
It looks like global issue. As example (read last comments): https://jukkaniiranen.com/2021/06/virtual-dataverse-tables-with-no-code-via-connectors/#comment-450134
